src/main/java/be/florens/eggtab/ItemGroupHandler.java

```diff
@@ -52,12 +52,12 @@ private static void hideGroupIfEmpty(ItemGroup itemGroup) {
 		}
 
 		((ItemGroupExtensions) itemGroup).eggtab$hideFromCreativeInventory();
-		EggTab.LOGGER.info("[Egg Tab] Removed an empty item group: " + itemGroup.getTranslationKey().getString());
+		EggTab.LOGGER.info("[Egg Tab] Removed an empty item group: " + itemGroup.getName());
```

Comment:
I think I used the translation key because this won't work on the dedicated server. Please double check what output this gives on client and server.

Reply:
`getTranslation()` was removed from ItemGroup but I've tested `getName()` and it works fine on dedicated servers.